KonyAccelerate/controllers/mobile/frmSponsorsController/frmSponsorsController.js

```diff
@@ -0,0 +1,11 @@
+define({ 
+
+  showSponsorsInPage:function(sponsors){
```

Comment:
Function comment is missing

Reply:
Removed the code, since we postponed to implement the sponsors' page later.